nerfstudio/model_components/losses.py

```diff
@@ -513,17 +513,18 @@ class GradientScaler(torch.autograd.Function):  # typing: ignore, pylint: disabl
     """
 
     @staticmethod
-    def forward(ctx, field_outputs, ray_dist):  # pylint: disable=arguments-differ
+    def forward(ctx, field_outputs, ray_samples):  # pylint: disable=arguments-differ
+        ray_dist = ray_samples.frustums.starts + ray_samples.frustums.ends / 2
```

Comment:
should be `(ray_samples.frustums.starts + ray_samples.frustums.ends) / 2`

Reply:
Thanks!